src/plugins/ui/evaluation_conditional_format.ts

```diff
@@ -188,7 +184,7 @@ export class EvaluationConditionalFormatPlugin extends UIPlugin {
         return percentile(rangeValues, Number(threshold.value) / 100, true);
       case "formula":
         const value = threshold.value && this.getters.evaluateFormula(threshold.value);
-        return !(value instanceof Promise) ? value : null;
+        return typeof value == "number" ? value : null;
```

Comment:
this is a behavior change, right ?
Technically, `value` could be a string before (even if it should not according to the method return type).
Are we missing a test where the formula evaluates to a string ?